.github/workflows/ci.yaml

```diff
@@ -441,20 +396,146 @@ jobs:
       - name: Build
         run: yarn build
 
+      - name: Build
+        run: yarn workspace integration-tests run build-and-pack
+
+      - name: Pull images
+        run: |
+          docker-compose -f integration-tests/docker-compose.yml pull \
+            db \
+            clickhouse \
+            zookeeper \
+            broker \
+            redis \
+            supertokens \
+            local_cdn \
+            external_composition
```

Comment:
@n1ru4l Why not just `pull`? Is there a reason?

Reply:
because it fail due to the other service images not being available (they need to be built locally).